src/main/flight/pid.c

```diff
@@ -415,7 +415,16 @@ void pidController(const pidProfile_t *pidProfile, const rollAndPitchTrims_t *an
             }
             if (axis == FD_YAW) {
                 // on yaw axis, prevent "yaw spin to the moon" after crash by constraining errorRate
-                errorRate = constrainf(errorRate, -crashLimitYaw, crashLimitYaw);
+                if (gyroRate > 1990.0f || gyroRate < -1990.0f) {
```

Comment:
Can we eliminate the magic number here?

Reply:
I thought about that. However the comment describing why this number is used directly follows the number itself. If we replaced the number with (say) a `#define` then we'd have to split or duplicate the comment. So understanding what was going on would require the reader to read comments in two places in the code.

Reply:
You could put the `#define` right ahead of the statement. That way you'll also get the positive and negative case covered by the same value.